### DISCUSSION TITLE
Customizing 3D models

### DISCUSSION THREAD

**Discussion by piyushaggarwal02** [google/model-viewer#1894] [Q&A]
Is there any way to customize the 3D model on browser or in AR via model-viewer?

My use case includes that I have to give the user an option to change model's textures on the browser and in AR via some button click event. So, is this possible in webxr supported browsers? (as I suppose Google's scene viewer & Apple's AR do not provide options to customize your 3D model)

And if not possible, is this in your pipeline?

**Comment by karanganesan** [comment]
> My use case includes that I have to give the user an option to change model's textures on the browser and in AR via some button click event.

Yes @piyushaggarwal02 take a look at 3DOM :)
https://github.com/google/model-viewer/tree/master/packages/3dom

> So, is this possible in webxr supported browsers? 

Yes

**Comment by elalish** [comment]
@piyushaggarwal02 Indeed, that functionality has just recently landed and is still beta, but please give it a try and give us feedback. We plan to expand this functionality going forward.

**Comment by piyushaggarwal02** [comment]
Sure. I will give it a try and share the feedback. Thanks
